File: cllc-public-app/ViewModels/OngoingLicenseeData.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System;
using System.Collections.Generic;

namespace Gov.Lclb.Cllb.Public.ViewModels
{

    public class OngoingLicenseeData
    {
        public Application Application { get; set; }
        public List<LicenseeChangeLog> ChangeLogs { get; set; }
        public int NonTerminatedApplications { get; set; }
        public LegalEntity CurrentHierarchy { get; set; }
        public List<ApplicationLicenseSummary> Licenses { get; set; }
    }
}

File: cllc-public-app/ClientApp/src/app/models/ongoing-licensee-data.ts
import { ApplicationLicenseSummary } from './application-license-summary.model';
import { Application } from './application.model';
import { LegalEntity } from './legal-entity.model';
import { LicenseeChangeLog } from './licensee-change-log.model';

export class OngoingLicenseeData {
  application: Application;
  changeLogs: LicenseeChangeLog[];
  nonTerminatedApplications: number;
  currentHierarchy: LegalEntity;
  licenses: ApplicationLicenseSummary[];
  treeRoot: LicenseeChangeLog;
}
